pkg/OWNERS

```diff
@@ -0,0 +1,4 @@
+files:
```

Comment:
You intentionally decided against a monolith file, right? We could have something at the repo root that has lines like

```
**/kv: kv
```

Reply:
PS the above pattern is one that I would use -- anything that has `kv` in it is owned by `kv`. (Or perhaps even: anything in a package starting with `kv` is). Curious what this would look like properly expressed. Will there be terrible regexps? I'd much rather deal with some of these `**` pattern matchers that were made for file matching.

Reply:
I chose the "spread out" model because one file can get way too big. not being able to fine grain by package without parsing a monolithic file can be quite gnarly for a human. keeping ownership as close to the package as possible is my preference.

Reply:
That said, since pkg/kv/OWNERS has

```
owners:
- kv
```

in it, which means that and its subdirectories are all KV.

Reply:
(thereoetically in this model, `**/kv` works in `pkg/OWNERS`, but again trying to avoid the monolithic file.